### ISSUE TITLE
Dark style of syntax and background

### ISSUE THREAD

**Issue by Shaddy** [eset/ipyida#7]
It's not really an issue but to all these guys that prefer a dark style :).

add to `qtconsole.py`

``` python
    from IPython.qt.console import styles
    ....
    self.ipython_widget.syntax_style = styles.default_dark_syntax_style
    self.ipython_widget.style_sheet = styles.default_dark_style_sheet
```

not sure if it's interesting to add a config interface or something, anyway for some reason the background is purely black instead of the grey monokai that ipython qtconsole have.

**Comment by Shaddy** [comment]
I'll answer myself on the "black" background.

You can add this in order to have a more realistic monokai style:

```python
self.ipython_widget.style_sheet = styles.default_dark_style_sheet.replace("black", "#272822")
```
I'm aware that it's not probably the best way to do it :)

**Comment by marc-etienne** [comment]
That's actually a neat idea. I'll try to implement it by copying the active stylesheet from IDA. That should make it in the next version.

**Comment by marc-etienne** [comment]
I was able to enable the dark theme using the following in my `ipyidarc.py`:

```
import qtconsole.styles
import ipyida.ida_qtconsole
ipyida.ida_qtconsole.set_widget_options(dict(
    style_sheet = qtconsole.styles.default_dark_style_sheet,
    syntax_style = qtconsole.styles.default_dark_syntax_style
))
```

This will be available in v1.4.

**Comment by marc-etienne** [comment]
v1.4 is now released.
